frontend/src/components/c2-event/index.tsx

```diff
@@ -0,0 +1,259 @@
+// Copyright 2020, Verizon Media
+// Licensed under the terms of the MIT. See LICENSE file in project root for terms.
```

Comment:
I think the majority of copyright headers have been removed, and we could probably drop the header. @jrozner  or @jkennedyvz can verify.

Reply:
That's correct @JoelAtDeluxe we no longer use copyright headers in each file.